checks/branch_protection.go

```diff
@@ -28,36 +28,56 @@ type branchProtectionSetting int
 
 const (
 	// CheckBranchProtection is the exported name for Branch-Protected check.
-	CheckBranchProtection                         = "Branch-Protection"
-	minReviews                                    = 2
-	allowForcePushes      branchProtectionSetting = iota
+	CheckBranchProtection = "Branch-Protection"
+	minReviews            = 2
+	// Points incremented at each level.
+	level1 = 3
+	level2 = 3
+	level3 = 2
+	level4 = 1
+	level5 = 1
+	// First level.
```

Comment:
could you map the levels to the names you've given them? 
1: (admin/non-admin) basic score
2. (admin/non-admin) review score
3. (non-admin) context
4. (non-admin) thorough review
5. (admin) thorough review

Reply:
sure

Reply:
sg, will do. consider it done.